app/src/main/java/org/jellyfin/mobile/utils/TickUtils.kt

```diff
@@ -0,0 +1,6 @@
+package org.jellyfin.mobile.utils
+
+object TickUtils {
+    fun ticksToMs(ticks: Long): Long = ticks / Constants.TICKS_PER_MS
+    fun msToTicks(ms: Long): Long = ms * Constants.TICKS_PER_MS
```

Comment:
We already have utilities to work with ticks via our SDK & Kotlin's Duration type.
Instead of doing `TickUtils.ticksToMs(someValue)` you can do `someValue.ticks.inWholeMilliseconds`.
for `msToTicks` that would be `.milliseconds.inWholeTicks`